docs/api-table.js

```diff
@@ -20,6 +20,24 @@ export const react = {
       ],
     ],
   },
+  Attention: {
+    required: [
+      [
+        'placement', "Directions", '', `Placement according to the target element Arrow would be on the opposite side of this position`,
```

Comment:
Having a bit of a hard time trying to understand the descriptions here. 😅 Missing periods?